app/cafe.py

```diff
@@ -0,0 +1,32 @@
+import datetime
+from app.errors import (
+    NotVaccinatedError,
+    OutdatedVaccineError,
+    NotWearingMaskError
+)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def __str__(self) -> str:
+        return self.name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        check = 0
+        if "vaccine" in visitor:
+            check += 1
+        if check == 0:

```

Comment:
It's a redundant variable, just use the condition `if "vaccine" not in visitor:`